tweaks/steam.yaml

```diff
@@ -1402,7 +1415,10 @@
 "212480": # Sonic & All-Stars Racing Transformed
   compat_tool: proton_63
 
-"71250": # Sonic Adventure DX
+"213610": # Sonic Adventure 2
+  compat_tool: proton_63
+
+"213610": # Sonic Adventure DX
```

Comment:
Oops, this is the app id for Sonic Adventure 2

Reply:
Sorry. C&P failure.